version.adoc

```diff
@@ -8,16 +8,18 @@ or with the commandline switch `-a final`.
 ////
 // :final:
 
-// No changes needed beyond this line
 ifdef::final[]
+// Change the DOI prior to releasing the next version
+:doi: 10.5281/zenodo.14275599
+// No changes needed beyond this line
 :current-version: {version}
 :current-version-as-attribute: {version}
-:doi: 10.5072/zenodo.FFFFFF
```

Comment:
don't remove it, ad here the actual pre-resrved DOI for v1.12. 

We will need to handle this better in the future

Reply:
Agreed. I moved the `:doi:` definition a few lines up, so it's still there.